src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -215,12 +215,24 @@ public void testEmptyCatchBlock() throws Exception {
     }
 
     @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/231")
```

Comment:
Please remove this ignore. It is not related to the current change for FallThrough.

Reply:
done